ios/Mattermost/AppDelegate.m

```diff
@@ -117,6 +118,13 @@ -(void)application:(UIApplication *)application didReceiveRemoteNotification:(no
   if (action && [action isEqualToString: NotificationClearAction]) {
     // If received a notification that a channel was read, remove all notifications from that channel (only with app in foreground/background)
     [self cleanNotificationsFromChannel:channelId andUpdateBadge:NO];
+    RuntimeUtils *utils = [[RuntimeUtils alloc] init];
+    [utils delayWithSeconds:0.2 closure:^(void) {
```

Comment:
I don't understand how it became `delayWithSeconds` when calling `delay` function in swift.  Can you give me some pointer about it?  thanks

Reply:
is the mapping that gets automatically created between swift and objective-C

delay = function name
With = helper created by the mapping
Seconds = name of the first argument

Reply:
Thanks!